src/App.tsx

```diff
@@ -1,44 +1,103 @@
-import React from 'react';
 import 'bulma/bulma.sass';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
 import classNames from 'classnames';
+import { useState, useEffect } from 'react';
 import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
+import { User } from './types/User';
+import { Post } from './types/Post';
+import { Comment } from './types/Comment';
+import { ErrorMessage } from './types/ErrorMessage';
+import { ShowLoader } from './types/ShowLoader';
+import { client } from './utils/fetchClient';
+
+const { LoadingComments, LoadingUser, LoadingPosts } = ErrorMessage;
 
 export const App: React.FC = () => {
+  const [activeUser, setActiveUser] = useState<User | null>(null);
+  const [posts, setPosts] = useState<Post [] | undefined>();
```

Comment:
Use an empty array for an initial value 
```suggestion
  const [posts, setPosts] = useState<Post []>([]);
```